lib/generators/koi/mailer/templates/model_template.rb

```diff
@@ -2,10 +2,29 @@ class <%= class_name %> < ActiveRecord::Base
 
   has_crud settings: true
 
+  <%- model_attributes.each do |attr| -%>
+      <%- if field_is_enum?(attr.name, attr.type) -%>
```

Comment:
Spacing is all over the place here. Can the block content not be replaced with <%= make_field_config(attr) -%>